### PR TITLE
Pr/54 - Added Bower file and updated package.json

### PR DESCRIPTION
**Pull request by TalAter**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  ],
+  "keywords": [
+    "offline",
+    "offlinefirst",
+    "cache",
+    "serviceworker"
+  ],
+  "license": "MIT",
+  "devDependencies": {
+    "grunt": "^0.4.5",
```
Comment (mckramer) [inline bower.json]: These are node dependencies, not bower dependencies and should be removed.

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "type": "git",
+    "url": "git://github.com/TalAter/UpUp.git"
+  }
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "Gruntfile.js",
+    "package.json",
+    "bower.json",
```
Comment (mckramer) [inline bower.json]: bower.json is never ignored, see: http://bower.io/docs/creating-packages/#ignore

**Comment by TalAter** [comment]
Thanks @mckramer. Made the changes you suggested.

**Comment by mckramer** [comment]
:+1:

**Pull request merged**